thrift/src/main/thrift/atoms/recipe.thrift

```diff
@@ -32,25 +32,37 @@ struct Range {
 struct Ingredient {
   1: required string item
   2: optional string comment
-  3: optional double quantity
+  // 3: optional double quantity
   4: optional Range quantityRange
   5: optional string unit
+  6: optional string dietaryInfo
+  7: optional Range caloryRange
+  8: required Range quantityRange
 }
 
+
 struct IngredientsList {
   1: optional string title
   2: required list<Ingredient> ingredients
 }
 
+struct RecipeStep {
+    1: required string name
+    2: required string text
+    3: optional shared.Image image
+    4: optional string link
+}
+
 struct RecipeAtom {
-  /* the unique ID will be stored in the `atom` data */
   1: required string title
-  2: required Tags tags 
+  2: required Tags tags
   3: required Time time
   4: optional Serves serves
   5: required list<IngredientsList> ingredientsLists
-  6: required list<string> steps
+  // 6: required list<string> steps
   7: required list<string> credits
   8: required list<shared.Image> images
   9: optional string sourceArticleId
-}
\ No newline at end of file
+  10: required string description
+  11: required list<RecipeStep> steps
```

Comment:
As `6: 6: required list<string> steps` has to stay, this would have to bear a new name that isn't `steps`. And again, consider carefully using anything that's `required` anyway as they will have to be back-filled in old data (so we'd have to reindex all Atoms with _something_ in those fields for the sake of the four or so that exist) and they can never be removed.

Reply:
Roger that, have addressed with [9b77085](https://github.com/guardian/content-atom/pull/156/commits/9b77085fd7b41818ea7ddfe22cefd43860a72c20)